### DISCUSSION TITLE
Is it possible to make Whim work on elevated/system windows

### DISCUSSION THREAD

**Discussion by Minoo7** [dalyIsaac/Whim#879] [Q&A]
Title (also including windows that are ran as administrator)

**Comment by dalyIsaac** [comment]
It might be possible to do by running Whim as an administrator. PowerToys' FancyZones has a similar issue - see https://learn.microsoft.com/en-us/windows/powertoys/administrator. However, I haven't tested this so there may be peculiarities about Whim's behavior.
